### PR TITLE
pr: modify the image address of busybox

### LINKED ISSUE #92: [GoInception] Failed to install goinception pod in my k8s in China region
**Describe the bug**
I cannot pull the image, busybox from docker hub since the installation of goinception is located in China region. As a result, the installation of the pod failed. (I tried to install the chart, Archery, also failed since the goinception is dependent part in the chart. )

I checked the deployment of goinception(see link below).  The address of busybox is hard coded. Can someone in team change it to an variable? Then I can modify the value to a local repo in China region instead of pointing to the docker hub directly. 

https://github.com/douban/charts/blob/3fa7828ca90be5e317e4a6c4ab9ca762e636e9df/charts/goinception/templates/deployment.yaml#L31

**To Reproduce**
Follow the https://github.com/douban/charts/tree/master/charts/archery
to install the Archery chart in China region, you will see the issue. 

**Expected behavior**
Want to install the goinception pod in China region successfully.

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by visla-xugeng**
fix #92

### CODE REVIEW

File: charts/goinception/values.yaml [hunk 1]
```diff
@@ -9,6 +9,12 @@ image:
   tag: latest
   pullPolicy: IfNotPresent
 
+override-configs:
```
Comment (LeoQuote) [inline charts/goinception/values.yaml:12]: 请用驼峰式命名风格

**Review by LeoQuote — CHANGES_REQUESTED** [review]
麻烦 bump version

File: charts/goinception/values.yaml (hunk 1 above)
Comment (visla-xugeng) [reply charts/goinception/values.yaml]: Just pushed the code again, please let me know if any questions.

**Review by LeoQuote — APPROVED** [review]

**Comment by LeoQuote** [comment]
麻烦关注下CI的结果，有一个行位的多余空格

**Pull request merged**